### PR TITLE
Clean up XDocument's Events tests

### PR DESCRIPTION
**Pull request by krwq**

### CODE REVIEW

File: src/System.Xml.XDocument/tests/events/EventsAdd.cs [hunk 1]
```diff
@@ -264,18 +219,18 @@ public void WorkOnTextNodes1()
                         using (EventsHelper eHelper = new EventsHelper(elem))
                         {
                             n.AddAfterSelf("text0");
-                            TestLog.Compare(elem.Value, "text2text0", "Did not concat text nodes correctly");
+                            Assert.Equal(elem.Value, "text2text0");
```
Comment (stephentoub) [inline src/System.Xml.XDocument/tests/events/EventsAdd.cs]: Anywhere we do Assert.Equal, the first argument should be the expected result and the second argument should be the actual one.

**Comment by stephentoub** [comment]
The improvements look good.  In addition to fixing the Equal thing I called out, can you also squash your part 1 - part 8 commits into a single commit?

File: src/System.Xml.XDocument/tests/events/EventsAdd.cs [hunk 2]
```diff
@@ -2,70 +2,30 @@
...
+        public partial class EventsTests
         {
-            //////////////////////////////////////////////////////////////////////////////////////////////////////
-            /// <summary>
-            /// EventsBaseClass: Base class that allows the dynamic addition of variations.  This makes it more
-            /// flexiable to add dynamic params to the variation.
-            /// </summary>
-            //////////////////////////////////////////////////////////////////////////////////////////////////////
-            public partial class EventsBase : XLinqTestCase
+            public class EventsAddBeforeSelf
```
Comment (Maxwe11) [inline src/System.Xml.XDocument/tests/events/EventsAdd.cs]: Why do we still need nested classes?

**Comment by krwq** [comment]
- Squashed commits with part 1-8,
- Replaced parameters
- Replaced nested classes with longer namespace
- Updated lock json

File: src/Common/tests/System/Xml/ModuleCore/project.lock.json [hunk 3]
```diff
@@ -1,5 +1,5 @@
 {
-  "locked": true,
+  "locked": false,
```
Comment (stephentoub) [inline src/Common/tests/System/Xml/ModuleCore/project.lock.json]: Same question as on the other PR about the locked value changing to false.

File: src/Common/tests/System/Xml/XmlCoreTest/project.lock.json [hunk 4]
```diff
@@ -1,5 +1,5 @@
 {
-  "locked": true,
+  "locked": false,
```
Comment (stephentoub) [inline src/Common/tests/System/Xml/XmlCoreTest/project.lock.json]: Ditto.

File: src/Common/tests/System/Xml/XmlDiff/project.lock.json [hunk 5]
```diff
@@ -1,5 +1,5 @@
 {
-  "locked": true,
+  "locked": false,
```
Comment (stephentoub) [inline src/Common/tests/System/Xml/XmlDiff/project.lock.json]: Ditto.  I'll stop commenting on these.

**Comment by stephentoub** [comment]
I skimmed through it, and other than the few comments, LGTM.

**Comment by krwq** [comment]
Removed lock.json commit

**Pull request merged**